rnndb/memory/g80_vm.xml

```diff
@@ -65,7 +65,7 @@
 </enum>
 
 <enum name="G80_STORAGE_TYPE" varset="chipset">
-	<value value="0x00" name="LINEAR"/>
+	<value value="0x00" name="PITCH"/>
```

Comment:
Kind of unfortunate naming (PITCH implies 2D use, while it's also used for all non-2D stuff), but let's match nvidia here.